shared/folders/render.js.flow

```diff
@@ -18,6 +18,9 @@ export type Props = {
   onOpen?: (path: string) => void,
   onRekey: (path: string) => void,
   onToggleShowIgnored: () => void,
+  installed: boolean,
+  fuseStatus: () => void,
```

Comment:
this should probably be called something like `getFuseStatus` otherwise it sounds like data vs function